src/js/constants/ServiceSchema.js

```diff
@@ -45,7 +45,7 @@ let SERVICE_SCHEMA = {
           type: 'number',
           default: 1,
           getter: function (service) {
-            return (service.getInstancesCount() || 0);
+            return (service.getInstancesCount() || 0) + '';
```

Comment:
Oh, confused now.
I don't think this "types" here are used to set the input field type.
There's is also a type of `object` and so on here in this scheme.
They are meant to be real javascript types. (Virtually)